### DISCUSSION TITLE
How do I center the page title on the homepage?

### DISCUSSION THREAD

**Discussion by phaethix** [imfing/hextra#267] [Q&A]
我想让这部分在页面的中间显示，而不是居左
![Snipaste_2024-01-21_23-37-56](https://github.com/imfing/hextra/assets/53891741/9ed59682-ea0a-447a-9733-185e0aaa4f6d)

**Comment by imfing** [comment]
> 我想让这部分在页面的中间显示，而不是居左
> "I want this part to be displayed in the middle of the page, not aligned to the left."

You may use HTML and CSS. Just wrap those in a div tag: `<div style="margin: auto; text-align: center;">`


```
<div style="margin: auto; text-align: center;">

<div class="mt-6 mb-6">
{{< hextra/hero-headline >}}
  Build modern websites&nbsp;<br class="sm:block hidden" />with Markdown and Hugo
{{< /hextra/hero-headline >}}
</div>

<div class="mb-12">
{{< hextra/hero-subtitle >}}
  Fast, batteries-included Hugo theme&nbsp;<br class="sm:block hidden" />for creating beautiful static websites
{{< /hextra/hero-subtitle >}}
</div>

<div class="mb-6">
{{< hextra/hero-button text="Get Started" link="docs" >}}
</div>

</div>
```

![image](https://github.com/imfing/hextra/assets/5097752/b29d02e2-4800-451e-acfd-d354605e81cc)

**Reply by phaethix** [reply]
Thank you very much for your help. I don't quite understand the front-end code, so I would like to ask how to center the copyright information below and reduce the distance between the text, that is, the distance between Powered by Hextra and the horizontal line above, because I want the page below to be compact and have a smaller font. Thank you very much.
![Snipaste_2024-01-22_10-18-53](https://github.com/imfing/hextra/assets/53891741/bf9f133d-b3f1-4f76-a67d-44306da29a36)

**Reply by imfing** [reply]
> Thank you very much for your help. I don't quite understand the front-end code, so I would like to ask how to center the copyright information below and reduce the distance between the text, that is, the distance between Powered by Hextra and the horizontal line above, because I want the page below to be compact and have a smaller font. Thank you very much.
> 
> ![Snipaste_2024-01-22_10-18-53](https://github.com/imfing/hextra/assets/53891741/bf9f133d-b3f1-4f76-a67d-44306da29a36)
> 
> 

In this case, you need to put similar tag to override the footer template.

https://github.com/imfing/hextra/blob/main/layouts/partials/footer.html

Or override the CSS using the custom CSS feature:

https://imfing.github.io/hextra/docs/advanced/customization/#custom-css

It might not be very easy if you don't know much about HTML and CSS. 

I'll keep this in mind and see if I could do something to help make it easier to customize the positions of these builtin elements.
